app/controllers/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+class UsersController < ApplicationController
+  def index; end
+
+  def singleuser; end
+
+  def userposts; end
+
+  def singleposts; end
```

Comment:
- [ ] We don't need all these actions. Kindly remove them.

- [ ] Kindly create the User `#show` action instead

  ```
  def show; end
  ```